mozilla/fx-private-relay
# Update to node 14.20.1

Update from node 14.18 / 14.19 to 14.20.1 (latest).

To test:
* Wait for CircleCI to run unit tests

macOS developers should update locally, for a [security update](https://nodejs.org/en/blog/vulnerability/september-2022-security-releases/) in 14.20.1.
